app/helpers/duplicate_intake_guard.rb

```diff
@@ -2,7 +2,7 @@ class DuplicateIntakeGuard < SimpleDelegator
   def has_duplicate?
     permitted_eip_only_values = eip_only ? true : [nil, false]
     Intake
-      .where.not(completed_at: nil)
```

Comment:
It seems so odd to be looking this up by a "string" value, but I guess it maps to an enum and thus an int in the db... It's definitely easier to read than where.not(primary_consented_to_service_at: nil)